### DISCUSSION TITLE
Nested Nodes

### DISCUSSION THREAD

**Discussion by keerthankrishna** [joyceworks/flowchart-react#34] [General]
Is it possible to have nested nodes using ` flowchart-react ` ? Please advice.
Here's an example - Please let me know if you want a diagrame to visualize this.
- You have one big node (_parent1_) within which there are 2 nodes connected to each other (_child1 and child2_).
- _parent1_ is then connected on the left to another parent node and on the right is connected to a different parent

**Comment by iamppz** [comment]
Nested node is not supported, but maybe I can add a dashed region instead of parent node
